### PR TITLE
Added an action server for the gripper

### PR DESCRIPTION
**Pull request by brennand**

### CODE REVIEW

File: kortex_description/grippers/robotiq_2f_85/urdf/robotiq_arg2f_85_model_macro.xacro [hunk 1]
```diff
@@ -176,7 +176,7 @@
       <parent link="${prefix}robotiq_arg2f_base_link" />
       <child link="${prefix}left_outer_knuckle" />
       <axis xyz="1 0 0" />
-      <limit lower="0" upper="0.8" velocity="2.0" effort="1000" />
+      <limit lower="0" upper="100" velocity="2.0" effort="1000" />
```
Comment (MarqRazz) [inline kortex_description/grippers/robotiq_2f_85/urdf/robotiq_arg2f_85_model_macro.xacro:179]: I can easily change the command interface to an angle (technically that is what it should be), I'm sorry you had to find all of these small fixes. Do you think that would be worth the effort?

File: kortex2_robotiq_gripper_driver/kortex2_robotiq_gripper_driver/robotiq_gripper_driver_85_action_server.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        return CancelResponse.ACCEPT
+
+
+    def _execute_callback(self, goal_handle):
+        self.get_logger().info('Gripper executing goal...')
+
+        # Send goal to gripper
+        cmd_msg = Float64MultiArray()
+
+        self.get_logger().warn("Commanded position: " + str(goal_handle.request.command.position))
```
Comment (MarqRazz) [inline kortex2_robotiq_gripper_driver/kortex2_robotiq_gripper_driver/robotiq_gripper_driver_85_action_server.py]: Can we make this a debug or info statement?

File: kortex_move_it_config/gen3_robotiq_2f_85_move_it_config/config/7dof/controllers.yaml [hunk 3]
```diff
@@ -15,3 +16,9 @@ joint_trajectory_controller:
     - joint_7
 
      # Gripper is currently on a command topic: /hand_controller/commands
```
Comment (MarqRazz) [inline kortex_move_it_config/gen3_robotiq_2f_85_move_it_config/config/7dof/controllers.yaml]: This is no longer needed.

**Review by MarqRazz — COMMENTED** [review]
This looks good to me (small comments below) but someone should try this on hardware tomorrow.

File: kortex_description/grippers/robotiq_2f_85/urdf/robotiq_arg2f_85_model_macro.xacro (hunk 1 above)
Comment (brennand) [reply kortex_description/grippers/robotiq_2f_85/urdf/robotiq_arg2f_85_model_macro.xacro:179]: Nope. all done now.

File: kortex2_robotiq_gripper_driver/kortex2_robotiq_gripper_driver/robotiq_gripper_driver_85_action_server.py (hunk 2 above)
Comment (brennand) [reply kortex2_robotiq_gripper_driver/kortex2_robotiq_gripper_driver/robotiq_gripper_driver_85_action_server.py]: I deleted it.  You where to quick to code review.

File: kortex2_bringup/config/gen3.srdf [hunk 4]
```diff
@@ -65,21 +65,21 @@
...
     <joint name="joint_7" value="0"/>
   </group_state>
-  <group_state group="gripper" name="opened">
-    <joint name="left_inner_finger_joint" value="0"/>
-    <joint name="right_inner_knuckle_joint" value="0"/>
-    <joint name="finger_joint" value="0"/>
-    <joint name="right_inner_finger_joint" value="0"/>
-    <joint name="left_inner_knuckle_joint" value="0"/>
-    <joint name="right_outer_knuckle_joint" value="0"/>
+  <group_state group="gripper" name="open">
```
Comment (cmelliott2) [inline kortex2_bringup/config/gen3.srdf]: 100 corresponds to closed and 0 corresponds to open, so the pose names need to be flipped

**Review by cmelliott2 — APPROVED** [review]

**Pull request merged**